docs/deployment/docker.md

```diff
@@ -85,6 +85,7 @@ WORKDIR /app
 # Copy files needed by Yarn
 COPY --chown=node:node .yarn ./.yarn
 COPY --chown=node:node .yarnrc.yml ./
+COPY --chown=node:node backstage.json ./
```

Comment:
@mtlewis do you think it would make sense and be possible to have the plugin gracefully handle a missing `backstage.json`? I'm thinking that it's just use whatever is in `yarn.lock` instead, all information you need would be there already right? Bumping the version is then more of a dev/CI-time thing where `backstage.json` is present, and the production build no longer requires it.